### DISCUSSION TITLE
How to prevent displaying page numbers on to: "odd" pagebreaks?

### DISCUSSION THREAD

**Discussion by chrissy0** [typst/typst#2337] [Q&A]
I want my page numbers to be displayed left/right based on the current page number, like this:
```
set page(
    footer: locate(loc => {
      let x = counter(page).at(loc).first()
      if calc.odd(x) [
        #align(right)[#counter(page).display()]
      ] else [
        #align(left)[#counter(page).display()]
      ]
  })
)
```

If I do `#pagebreak()#pagebreak()`, the empty page doesn't contain any page number. That's good. If I do `#pagebreak(to: "odd")` the empty page does contain a page number. I don't want this.

How do I achieve empty pages without page numbers without manually skipping multiple pages if necessary?

Thanks.

**Comment by yochem** [comment]
If you want empty pages without page numbers, consider this instead of `#pagebreak()`:

```typst
Page 1 content...

#page(footer: none)[] // empty page 2

Page 3 content...
```

Does that answer your question?

**Comment by Julian-Wassmann** [comment]
I`ve anwsered this in a different question: #3122
